.clitest/__init__.zsh

```diff
@@ -2,4 +2,5 @@ zmodload zsh/param/private
 . scripts/opts.zsh
 setopt ${_znap_opts:#warn*}
 builtin autoload -Uz $PWD/functions/{znap,(|.).znap.*~*.zwc}
+builtin autoload -Uz $PWD/.clitest/functions/.test.*~*.zwc
```

Comment:
Please don't autoload the test functions in production. 🙂 Move this code to somewhere where they get loaded only when testing.

Reply:
Can you clarify? The `./.clitest/__init__.zsh` file is global, and not just for initializing tests? That seems like it's in the wrong folder then.

Reply:
Ah, sorry, I was confusing it with [the other `__init__`](https://github.com/marlonrichert/zsh-autocomplete/blob/main/scripts/.autocomplete.__init__).